docs/examples/simulation.md

```diff
@@ -0,0 +1,392 @@
+
+
+```python
+# create a Computation node
+init = cript.Computation(
+    name="Initial snapshot and force-field generation",
+    type="initialization",
+    software_configuration=[
+        python_config,
+        rdkit_config,
+        stage_config,
+        packmol_config,
+        openmm_config,
+    ],
+)
+
+# create a computation node with Condition nodes
+equi = cript.Computation(
+    name="Equilibrate data prior to measurement",
+    type="MD",
+    software_configuration=[python_config, openmm_config],
+    condition=[
+        cript.Condition(key="time_duration", type="value", value=100.0, unit="ns"),
+        cript.Condition(key="temperature", type="value", value=450.0, unit="K"),
+        cript.Condition(key="pressure", type="value", value=1.0, unit="bar"),
+        cript.Condition(key="number", type="value", value=31),
+    ],
+    prerequisite_computation=init,
+)
+
+bulk = cript.Computation(
+    name="Bulk simulation for measurement",
+    type="MD",
+    software_configuration=[python_config, openmm_config],
+    condition=[
+        cript.Condition(key="time_duration", type="value", value=50.0, unit="ns"),
+        cript.Condition(key="temperature", type="value", value=450.0, unit="K"),
+        cript.Condition(key="pressure", type="value", value=1.0, unit="bar"),
+        cript.Condition(key="number", type="value", value=31),
+    ],
+    prerequisite_computation=equi,
+)
+
+ana = cript.Computation(
+    name="Density analysis",
+    type="analysis",
+    software_configuration=[python_config],
+    prerequisite_computation=bulk,
+)
+
+# Add all these computations to the experiment.
+experiment.computation += [init, equi, bulk, ana]
+```
+
+
+!!! note "Computation types"
+    The allowed `Computation` types are listed under <a href="https://criptapp.org/keys/computation-type/" target="_blank">computation types</a> in the CRIPT controlled vocabulary.
+
+!!! note "Condition keys"
+    The allowed `Condition` keys are listed under <a href="https://criptapp.org/keys/condition-key/" target="_blank">condition keys</a> in the CRIPT controlled vocabulary.
+
+
+# Create and Upload Files
+
+New we'd like to upload files associated with our simulation. First, we'll instantiate our File nodes under a specific project.
+
+```python
+packing_file = cript.File("Initial simulation box snapshot with roughly packed molecules", type="computation_snapshot", source="path/to/local/file")
+forcefield_file = cript.File(name="Forcefield definition file", type="data", source="path/to/local/file")
+snap_file = cript.File("Bulk measurement initial system snap shot", type="computation_snapshot", source="path/to/local/file")
+final_file = cript.File("Final snapshot of the system at the end the simulations", type="computation_snapshot", source="path/to/local/file")
+```
+
+!!! note
+The `source` field should point to any file on your local filesystem.
```

Comment:
user can put source as `https://en.wikipedia.org/wiki/John_von_Neumann` and that is good and valid, so we are adaptable for the users in case they want to use their own solutions for storage

Reply:
see, comment above. I def. want a local file path here.

The manually specified remote file is a special case, that is best found a specialized place, not a general introduction.

Reply:
good point! agreed!